pages/docs/reference/whatsnew14.md

```diff
@@ -35,6 +41,288 @@ fun main(){
+
+```kotlin
+// FILE: test.kt
+fun test(r1: Runnable) {
+    A.foo(r1) {}  // Works in Kotlin 1.4
+}
+```
+
+</div>
+
+#### Java SAM interfaces in Kotlin
+
+In Kotlin 1.4, you can use Java SAM interfaces in Kotlin and apply SAM conversions to them. 
+In Kotlin 1.3, you could perform a SAM conversion only if you declared such a function `foo` in Java code.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+import java.lang.Runnable
+
+fun foo(r: Runnable) {}
+
+fun test() { 
+    foo { } // OK
+}
+```
+
+</div>
+
+### Callable reference improvements
+
+Kotlin 1.4 supports more cases for using callable references:
+
+* References to functions with default argument values
+* Function references in `Unit`-returning functions 
+* References adapting to functions with a variable number of arguments (`vararg`)
+* Suspend conversion on callable references 
+
+#### References to functions with default argument values 
+
+Now you can use callable references to functions with default argument values. If the callable reference 
+to the function `foo` takes no arguments, the default value `0` is used.
+
+<div class="sample" markdown="1" theme="idea" data-min-compiler-version="1.4">
+
+```kotlin
+fun foo(i: Int = 0): String = "$i!"
+
+fun apply(func: () -> String): String = func()
+
+fun main() {
+    println(apply(::foo))
+}
+```
+
+</div>
+
+Previously, you should write additional overloads for the function `apply` to use the default argument values.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+// some new overload
+fun applyInt(func: (Int) -> String): String = func(0) 
+```
+
+</div>
+
+#### Function references in `Unit`-returning functions 
+
+In Kotlin 1.4, you can use callable references to functions returning any type in `Unit`-returning functions. 
+Before Kotlin 1.4, you could use only lambda arguments in this case. Now you can use both.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+fun foo(f: () -> Unit) { }
+fun returnsInt(): Int = 42
+
+fun main() {
+    foo { returnsInt() } // it was OK before 1.4
+    foo(::returnsInt) // since Kotlin 1.4, it works as well
+}
+```
+
+</div>
+
+#### References adapting to functions with a variable number of arguments
+
+Now you can adapt callable references to functions when passing a variable number of arguments (`vararg`) . 
+You can pass any number of parameters of the same type at the end.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+fun foo(x: Int, vararg y: String) {}
+
+fun use0(f: (Int) -> Unit) {}
+fun use1(f: (Int, String) -> Unit) {}
+fun use2(f: (Int, String, String) -> Unit) {}
+
+fun test() {
+    use0(::foo) 
+    use1(::foo) 
+    use2(::foo) 
+}
+```
+
+</div>
+
+#### Suspend conversion on callable references
+
+In addition to suspend conversion on lambdas, Kotlin 1.4 supports suspend conversion on callable references in Kotlin/JVM.
```

Comment:
> callable references in Kotlin/JVM

Why only `Kotlin/JVM`? As I know, this kind of conversion works everywhere

Reply:
Removed.